LiskHQ/lisk-hub
# Update the desktop app icon - Closes #495

### What was the problem?
icons for desktop app are outdated and need to be updated
### How did I fix it?
replace assets in project
### How to test it?
see new generated ios bundle, install desktop app, check for icons in:

- DMG Install dialog 
- Spotlight search
- Applications bottom Bar
- Tab view on active applications
- Applications folder

### Review checklist
- The PR solves #495 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
